adapters/adot/adot_test.go

```diff
@@ -75,3 +75,11 @@ func TestMediaTypeForBid(t *testing.T) {
 		t.Errorf("the type is not the valid one. actual: %v, expected: %v", bidTypeNative, openrtb_ext.BidTypeVideo)
 	}
 }
+
+//Test macros resolution for adm and nurl
```

Comment:
Nitpick, I would say the comments above this test function, along with the others in this file aren't necessary. The name of the test function itself describes what the test is doing well enough!

Reply:
They were already there so i kept the same norm, but i understand your point.
I removed them.